components/create_comment/create_comment.jsx

```diff
@@ -184,6 +186,11 @@ class CreateComment extends React.PureComponent {
          * The last time, if any, when the selected post changed. Will be 0 if no post selected.
          */
         selectedPostFocussedAt: PropTypes.number.isRequired,
+
+        /**
+         * Functiont to set or unset emoji picker for last message
```

Comment:
```suggestion
         * Function to set or unset emoji picker for last message
```

Please also check other comments with spelling error